test/bases/renku_data_services/data_api/test_schemathesis.py

```diff
@@ -93,10 +93,7 @@ def filter_query(context: HookContext, query: dict[str, str] | None) -> bool:
 ]
 
 # TODO: RE-enable schemathesis when CI setup for notebooks / sessions is ready
-EXCLUDE_PATH_PREFIXES = [
-    "/sessions",
-    "/notebooks",
-]
+EXCLUDE_PATH_PREFIXES = ["/sessions", "/notebooks", "/renku_v1_projects"]
```

Comment:
question: why are we excluding the endpoint from schemathesis here?

Reply:
Fixed, previously excluded by mistake.